LINUX/Makefile

```diff
@@ -164,6 +164,9 @@ uninstall: $(call CREATE_TARGET,$(SUBDIRS_ALL_NON_OPTIONAL) $(SUBDIRS_OPTIONAL),
 	ldconfig || /bin/true
 
 ldconfig:
+	if [ ! -d "${DESTDIR}/etc/ld.so.conf.d" ]; then \
+	   mkdir "${DESTDIR}/etc/ld.so.conf.d"; \
```

Comment:
Wouldn't a "mkdir -p" be better, just in case that $(DESTDIR)/etc does not exist (yet)?

Reply:
Yes, indeed.